content/en/docs/contribute/participate/issue-wrangler.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Issue Wranglers
+content_type: concept
+weight: 20
+---
+
+<!-- overview -->
+
+In order to reduce the burden on the [PR Wrangler](/docs/contribute/participate/pr-wrangler), formal approvers, and reviewers, members of SIG Docs take week long shifts [triaging and categorising issues](/docs/contribute/review/for-approvers.md/#triage-and-categorize-issues) for the repository.
+
+<!-- body -->
+
+## Duties
+
+Each day in a week-long shift as Issue Wrangler:
+
+- Triage and tag incoming issues daily. See [Triage and categorize issues](https://github.com/kubernetes/website/blob/main/docs/contribute/review/for-approvers/#triage-and-categorize-issues) for guidelines on how SIG Docs uses metadata.
```

Comment:
The github link appears to be directing to a 404 error page; an update might be needed. 
_(There is also option to use https://kubernetes.io/docs/contribute/review/for-approvers/#triage-and-categorize-issues)_

 ```suggestion
- Triage and tag incoming issues daily. See [Triage and categorize issues](https://github.com/kubernetes/website/blob/main/content/en/docs/contribute/review/for-approvers.md#triage-and-categorize-issues) for guidelines on how SIG Docs uses metadata.
```